docs/Project.toml

```diff
@@ -1,2 +1,3 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+PowerModelsRestoration = "23adbb12-a187-11e9-26a2-eb4d4e6e68fb"
```

Comment:
I don't think we need this, seems to be working without this in other packages.

Reply:
I needed this and load path to build the docs locally.  Must be a windows thing?  I will remove.